### DISCUSSION TITLE
Is it possible to dismiss a V-Banner in Vuetify 3?

### DISCUSSION THREAD

**Discussion by robertpatrick** [vuetifyjs/vuetify#18187] [General] [answered]
V-Banner claims to support a `dismiss` function in the `actions` slots (see https://vuetifyjs.com/en/api/v-banner/#slots-actions) but the function does not seem to exist (or I am just not smart enough to understand how to use it).  

```html
    <template v-slot:actions>
      <v-btn
        variant="text"
        @click="dismiss()"
      >
        Dismiss
      </v-btn>
    </template>
```

results in a runtime error when clicking the button.

<img width="985" alt="Screenshot 2023-08-30 at 4 11 26 PM" src="https://github.com/vuetifyjs/vuetify/assets/31662131/6e01e9a9-a059-490c-8d76-9fcd8dda52f6">

Does this functionality exist or not?

**Comment by KaelWD** [comment, marked as answer]
It would be `v-slot:actions="{ dismiss }"`

The documentation is incorrect though, that function does not exist.
